package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engines": {
     "yarn": "^1.21.1",
-    "node": ">=12.13"
+    "node": ">=16.14"
```

Comment:
🔥

Reply:
Welcome to the future :D

Reply:
I'm excited about this, but also curious what impact this has on our local development. Some libraries don't play well with v16 yet (but v14 is ok). Is this considered breaking? Should we generally announce to ui devs that we should all upgrade to Node 16?

Reply:
@shawyu-okta yeah, for now we are slowly updating only our UI repos to v16 and fixing any breaking libraries/tests. We probably would have to switch between node versions using nvm based on the repos until we update all of our repos to node 16.

Reply:
Can/should we add `.nvmrc` files to each repo? (not sure if this has been discussed before)

It will make picking the right node version simpler if we can do `nvm use` when changing repositories.

Reply:
@shawyu-okta that's a great idea. I'll add it in a follow-up PR.